### PR TITLE
CLD-3772 Add support for max connections alarms

### PR DESCRIPTION
**Pull request by stylianosrigas**
#### Summary
- Add support for max connections alarms

#### Release Note

```release-note
- Add support for max connections alarms
```

### CODE REVIEW

**Review by d-wierdsma — APPROVED** [review]

**Review by stafot — APPROVED** [review]
Nice!

File: main.go [hunk 1]
```diff
@@ -45,6 +45,20 @@ var DBInstanceClassMemory = map[string]string{
 	"db.r5.24xlarge": "824633720832",
 }
 
+// DBInstanceClassConnections maps DB instance types with their max connections
+var DBInstanceClassConnections = map[string]string{
```
Comment (spirosoik) [inline main.go:49]: is there an API in AWS sdk which gives these numbers?

File: main.go [hunk 2]
```diff
@@ -75,7 +89,23 @@ var DBInstanceGravitonClassMemory = map[string]string{
 	"db.r6g.24xlarge": "824633720832",
 }
 
+// DBInstanceGravitonClassConnections maps DB Graviton instance types with their max connections
+var DBInstanceGravitonClassConnections = map[string]string{
```
Comment (spirosoik) [inline main.go:93]: same for this

**Review by spirosoik — APPROVED** [review]
just one minor comment

**Review by angeloskyratzakos — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (stylianosrigas) [reply main.go]: Nope unfortunately :/

**Pull request merged**